docs/security/authorization/spark/install.md

```diff
@@ -92,6 +92,16 @@ Macros are now supported for using user/group/tag in row filter expressions, int
     </property>    
 ```
 
+##### Checking all required Access privileges
```

Comment:
the naming and meaning of this configuration are ambiguous.

No matter whether it is enabled or disabled, we need to check all required privileges.

Reply:
Changed to `Showing all disallowed privileges`.